training_record/tests/tests_serializers.py

```diff
@@ -33,3 +34,15 @@ def test_validate_attachments_data_attachments_too_large(
                 serializers.ValidationError,
                 '上传附件过大，请修改后再上传。(附件大小: 100 MB)'):
             serializer.validate_attachments_data(data)
+
+
+class TestCampusEventFeedbackSerializer(TestCase):
+    '''Unit tests for serializer of CampusEventFeedback.'''
+    @patch('training_record.serializers.CampusEventFeedbackService')
+    def test_create(self, mocked_service):
+        '''Should create feedback'''
+        serializer = CampusEventFeedbackSerializer()
+
+        mocked_service.create_feedback.return_value = '1'
+
+        self.assertEqual(serializer.create({}), '1')
```

Comment:
You should assert that `mocked_service.create_feedback` to have been called instead of assertion on a return value.